core/chains/evm/txmgr/attempts.go

```diff
@@ -75,11 +76,41 @@ func (c *evmTxAttemptBuilder) NewBumpTxAttempt(ctx context.Context, etx Tx, prev
 	if err != nil {
 		return attempt, bumpedFee, bumpedFeeLimit, true, pkgerrors.Wrap(err, "failed to bump fee") // estimator errors are retryable
 	}
-
+	// If transaction's previous attempt is marked for purge, ensure the new bumped attempt also sends empty payload, 0 value, and LimitDefault as fee limit
+	if previousAttempt.IsPurgeAttempt {
+		etx.EncodedPayload = []byte{}
+		etx.Value = *big.NewInt(0)
```

Comment:
We do set it in `NewPurgeTxAttempt` on [line 104](https://github.com/smartcontractkit/chainlink/pull/12881/files/31d69baf178c21cc8f675270d5020bac1e648efc#diff-5fddc44b0ab74c9628d9713258d13140d521eec25f481719f5e6ad76d6bc27c9R104)

Reply:
Since we derive the attempt from `etx`'s fields, I think I have to set it to empty here again. I didn't want to clear the payload or value of the tx in the DB for better tracking/logging so these fields are populated on bump too.